client/draft_README.md

```diff
@@ -366,19 +366,19 @@ Gratitude to the following people who have contributed to this project:
 </div>
 
 <h3>Sergio Silva</h3>
-<a href="mailto:[email]@[provider]" target="_blank">
+<a href="mailto:sergiobtos@hotmail.com" target="_blank">
     <img src="https://img.shields.io/badge/-[email_address]-c14438?style=flat&logo=Gmail&logoColor=white">
  </a>
 <div style='display: flex; justify-content: center; gap: .5rem'>
 
-<a href="https://www.linkedin.com/in/username" target="_blank">
+<a href="https://www.linkedin.com/in/sergio-btos" target="_blank">
       <img src="https://img.shields.io/badge/Sergio Silvay-blue?style=flat&logo=Linkedin&logoColor=f32dc8">
  </a>
 
 [@sergiobtos](https://github.com/sergiobtos)
 
-<a href="https://[username]g.github.io/portfolio" target="_blank">
-    <img src="https://img.shields.io/badge/[username]github.io/portfolio-gold?style=flat&logo=github"
+<a href="https://github.com/sergiobtos" target="_blank">
+    <img src="https://img.shields.io/badge/sergiobtos.github.io/portfolio-gold?style=flat&logo=github"
```

Comment:
Hi @sergiobtos!  The previous link directs to your GitHub profile.  The intention for this line/link is to refer to your portfolio website if you have one that you want to share.  Do you have a link you would like to share?  If so, please update.

Reply:
@CodeMeKathy  I dont have any for now, my portfolio website is my git hub, thats why I use github url. I wont create any portfolio website this year. Maybe in the future, but not now.